<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>渲染大作业 By Sun</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="image">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { background: gray; }</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockScene">
   <property name="windowTitle">
    <string>对象树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="contentObject">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="objectTree">
       <property name="indentation">
        <number>10</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="objectControls">
       <item>
        <widget class="QPushButton" name="objectAddGroup">
         <property name="maximumSize">
          <size>
           <width>46</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:enabled { color: green; }</string>
         </property>
         <property name="text">
          <string>+ 组</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="objectClearSelection">
         <property name="maximumSize">
          <size>
           <width>72</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>清除选择</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="objectControlSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="objectAdd">
         <property name="maximumSize">
          <size>
           <width>23</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:enabled { color: green; }</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="objectRemove">
         <property name="maximumSize">
          <size>
           <width>23</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:enabled { color: red; }</string>
         </property>
         <property name="text">
          <string>×</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="objectProperties">
       <property name="title">
        <string>对象属性</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="objectNameLabel">
          <property name="text">
           <string>名字：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="objectName_"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="objectVisibleLabel">
          <property name="text">
           <string>可见：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="objectVisible">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="objectMaterialLabel">
          <property name="text">
           <string>材料：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="objectMaterial">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="shapeProperties">
       <property name="title">
        <string>形状属性</string>
       </property>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="shapeTypeLabel">
          <property name="text">
           <string>类型：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="shapeType">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockMaterial">
   <property name="windowTitle">
    <string>材料</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="contentMaterial">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListView" name="materialList"/>
     </item>
     <item>
      <widget class="QGroupBox" name="materialProperties">
       <property name="title">
        <string>属性</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockTexture">
   <property name="windowTitle">
    <string>质地</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="contentTexture">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListView" name="textureList"/>
     </item>
     <item>
      <widget class="QGroupBox" name="textureProperties">
       <property name="title">
        <string>属性</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockCamera">
   <property name="windowTitle">
    <string>相机</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="contentCamera">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="cameraProperties">
       <property name="title">
        <string>相机属性</string>
       </property>
       <layout class="QFormLayout" name="formLayout_4"/>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="rendererProperties">
       <property name="title">
        <string>渲染属性</string>
       </property>
       <layout class="QFormLayout" name="formLayout_5"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockOperation">
   <property name="windowTitle">
    <string>操作记录</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="contentOperation">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QListView" name="operationList"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
